clients/admin-ui/src/pages/messaging/index.tsx

```diff
@@ -0,0 +1,275 @@
+          cell: (props) => (
+            <GroupCountBadgeCell
+              suffix="properties"
+              value={props.getValue()}
+              {...props}
+            />
+          ),
+          header: (props) => (
+            <DefaultHeaderCell value="Properties" {...props} />
+          ),
+          meta: {
+            displayText: "Properties",
+            showHeaderMenu: true,
+          },
+          size: 250,
+        }
+      ),
+      columnHelper.accessor((row) => row.is_enabled, {
+        id: "is_enabled",
+        cell: (props) => (
+          <Flex align="center" justifyContent="flex-start" w="full" h="full">
+            <Switch
+              isChecked={props.getValue()}
+              colorScheme="complimentary"
+              onChange={async (e) => {
+                const isEnabled = e.target.checked;
+
+                const result = await patchMessagingTemplateById({
+                  templateId: props.row.original.id,
+                  template: { is_enabled: isEnabled },
+                });
+
+                if (isErrorResult(result)) {
+                  toast(
+                    errorToastParams(
+                      `This message cannot be enabled because another message already exists with the same configuration. Change the property to enable this message.`
+                    )
+                  );
+                  return;
+                }
+
+                toast(
+                  successToastParams(
+                    `Messaging template ${isEnabled ? "enabled" : "disabled"}`
+                  )
+                );
+              }}
+            />
+          </Flex>
+        ),
+        header: (props) => <DefaultHeaderCell value="Enable" {...props} />,
+        size: 200,
+        meta: {
+          disableRowClick: true,
+        },
+      }),
+    ],
+    [toast, patchMessagingTemplateById]
+  );
+
+  const sortedData = sortBy(data, "id");
+  const tableInstance = useReactTable<MessagingTemplateWithPropertiesSummary>({
+    getCoreRowModel: getCoreRowModel(),
+    getFilteredRowModel: getFilteredRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    columns,
+    data: sortedData,
+  });
+
+  return (
+    <FixedLayout
+      title="Messaging"
+      mainProps={{
+        padding: "0 40px 10px",
+      }}
+    >
+      <PageHeader breadcrumbs={[{ title: "Messaging" }]}>
+        <Text fontWeight={500} color="gray.700">
+          Configure Fides messaging.
+        </Text>
+      </PageHeader>
+
+      <TableActionBar>
+        <HStack alignItems="center" spacing={4} marginLeft="auto">
+          <Button
+            size="xs"
+            colorScheme="primary"
+            data-testid="add-message-btn"
+            onClick={() => setIsAddTemplateModalOpen(true)}
+          >
+            Add message +
+          </Button>
+        </HStack>
+      </TableActionBar>
+
+      {isLoading && <TableSkeletonLoader rowHeight={36} numRows={15} />}
+      {!isLoading && (
```

Comment:
I know we're pro-`&&` for components now, but it feels weird for this to be two separate conditions and not a ternary.

Reply:
Updated to a ternary. However, I would find a ternary hard to read if both conditions had large components inside (not in this case).